banking/klarna_kosma_integration/doctype/banking_settings/banking_settings.py

```diff
@@ -89,7 +89,9 @@ def intraday_sync_ebics():
 		)
 
 	banking_settings = frappe.get_single("Banking Settings")
-	if not banking_settings.enabled or not banking_settings.enable_ebics:
+	if (not banking_settings.enabled and not banking_settings.enable_ebics) or frappe.conf.get(
```

Comment:
```suggestion
	if (not banking_settings.enabled or not banking_settings.enable_ebics) or frappe.conf.get(
```

Diesen Teil der Bedingung zu ändern war nicht vorgesehen